### PR TITLE
Task 6

### PR DESCRIPTION
**Pull request by zsummy**
All is done unless test and filter policy. Unfortunately customer's deadlines are hard and in addition I have a LIFE :)
5/7

### CODE REVIEW

**Comment by kotmatpockuh** [comment]
I would say 4/6 🤔

File: product-service/package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
   "dependencies": {
     "@types/joi": "^14.3.4",
     "@types/pg": "^7.14.6",
+    "aws-sdk": "^2.797.0",
```
Comment (kotmatpockuh) [inline product-service/package.json:20]: move it to devDeps to reduce bundle size ;)

**Comment by zsummy** [comment]
> I would say 4/6 🤔

Yeah, sorry for confusion

File: product-service/package.json (hunk 1 above)
Comment (zsummy) [reply product-service/package.json:20]: Thanks for pointing out, I forgot to optimise product-service bundles:)
However I think it's better to split deps types without considering bundle size. 'aws-sdk' is not a dev dependency, it's needed for the service to work :)
I prefer `externals` in webpack config:
https://github.com/zsummy/nodejs-aws-be/blob/119d31d15804050bb1ddf580fe3374ac57fd18fa/import-service/webpack.config.js#L23

**Pull request merged**